src/Analyzers/Core/Analyzers/AnalyzersResources.resx

```diff
@@ -325,4 +325,7 @@
   <data name="Remove_unnecessary_suppression" xml:space="preserve">
     <value>Remove unnecessary suppression</value>
   </data>
+  <data name="Convert_typeof_to_nameof" xml:space="preserve">
+    <value>Convert 'typeof' to 'nameof'</value>
```

Comment:
note, we will likely need a specific string for each langauge. for example, in vb this would be `Convert 'GetType' to 'NameOf'`.

Reply:
comment still applies.